tracing.compose.yaml

```diff
@@ -3,15 +3,15 @@ version: "3.8"
 services:
   buildkitd:
     container_name: dagger-buildkitd-jaeger
-    image: moby/buildkit:v0.8.3
+    image: moby/buildkit:latest
```

Comment:
I changed it to latest because, as you can see, we forgot to update it since v0.8.3 :)

There's a slight chance we'll run a version of buildkit that's too new. But I think there's a bigger chance we'll forget to update this file and run a very old version.